rss2017/process/officialReviewProcess.js

```diff
@@ -0,0 +1,38 @@
+function(){
+    var or3client = lib.or3client;
+
+    const CONFERENCE = 'roboticsfoundation.org/RSS/2017/Workshop';
+    const COCHAIRS = CONFERENCE+"/Program_Co-Chairs"
+    const PAPERGRP = CONFERENCE+'/Paper' + note.number;
+    const PAPERINV = CONFERENCE+'/-/Paper' + note.number;
+    const DUE_DATE = new Date(2017, 6, 30, 17, 15);
+
+    console.log('PAM - the official Review Process is here');
```

Comment:
these are actually sometimes useful for debugging - since we're not done setting up this conference yet, can we keep them for now?